tools/obj2vol/main.c

```diff
@@ -54,7 +54,7 @@ static bool _load_obj( const char* filename ) {
   if ( !v_buff || !vn_buff || !vt_buff || !f_buff ) { return false; }
 
```

Comment:
I forgot to make this multi-platform. It's a tiny tool to (badly) convert small .objs (like a cube) into volograms for testing.
The easy option here might be just to revert this change.